script/DeployUSDC.s.sol

```diff
@@ -20,12 +19,12 @@ contract DeployUSDCScript is Script {
     uint8 constant MAINNET_USDC_DECIMALS = 6;
 
     // PriceFeed config
-    address constant MAINNET_USDC_ETH_CHAINLINK = 0x986b5E1e1755e3C2440e960477f25201B0a8bbD4;
-    uint256 constant USDC_ETH_CHAINLINK_HEARTBEAT = 24 hours;
-    uint256 constant PRICE_UPPER_BOUND = 0.01e18; // i.e. 100 tokens buy 1 ETH
-    uint256 constant PRICE_LOWER_BOUND = 0.00001e18; // i.e. 100K tokens buy 1 ETH
+    address constant MAINNET_ETH_USDC_CHAINLINK = 0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419;
```

Comment:
maybe good to rename to ETH_USD (drop the C)

Reply:
fixed